src/appkit.cs

```diff
@@ -2109,10 +2109,12 @@ interface NSButtonCell {
 		[Export ("keyEquivalentModifierMask")]
 		NSEventModifierMask KeyEquivalentModifierMask { get; set; }
 	
-		[Export ("keyEquivalentFont", ArgumentSemantic.Retain)]
+		[NullAllowed, Export ("keyEquivalentFont", ArgumentSemantic.Strong)]
+		[Deprecated (PlatformName.MacOSX, 10, 15, message: "The keyEquivalentFont property is no longer used. It always returns the NSButtonCell's font, and setting it has no effect.")]
```

Comment:
Our API is named `KeyEquivalentFont` (casing) and it's also not formatted correctly. Also you cannot copy/paste messages from the header files.